build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     compile 'org.apache.maven:maven-model-builder:3.0.4'
+    compile 'commons-codec:commons-codec:1.10'
```

Comment:
So we want as few dependencies as possible. I believe the jdk or gdk can do this base64 encode without pulling in another library